prow/plugins/help/help.go

```diff
@@ -93,6 +93,11 @@ func handleGenericComment(pc plugins.PluginClient, e github.GenericCommentEvent)
 }
 
 func handle(gc githubClient, log *logrus.Entry, cp commentPruner, e *github.GenericCommentEvent) error {
+	// Only consider open issues and new comments.
+	if e.IsPR || e.IssueState != "open" || e.Action != github.GenericCommentActionCreated {
```

Comment:
Will you add unit tests for these new cases?
* Ignore a PR
* Ignore a closed issue
* Ignore a non-created comment

Reply:
Done.